catkin_ws/src/state_estimation/launch/hydrophones.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+     <!-- Distances to hydrophone 1 -->
+	<param name="hydrophones_dx" value="0.1"/>
+	<param name="hydrophones_dy" value="0.1"/>
+	<param name="hydrophones_dz" value="-0.1"/>
+	<!-- Frequency of each pinger -->
+	<!-- <param name="pinger_frequency_1" value="1"/>
```

Comment:
can just remove rather than comment out

Reply:
although actually why is this commented out isnt it still used in the bearing calculation file

Reply:
I think it makes more sense to be in hydrophones than sensor status

Reply:
yeah, I just did. The params are loaded in sensor_status.launch

Reply:
not sure if u saw the comment, it makes more sense to have pinger frequencies in hydrophones.launch no? If we want to launch hydrophones without sensor status / sensors currently it will crash. Neither should crash, but sensor status can be launched without hydrophones and that makes sense: so we can just add a try/catch on the parameter getting with maybe a while loop so that it crashes elegantly and with a clear error message

Reply:
but then we would have to launch state_estimation before sensor_status, because they both use the same frequency param